prometheus/src/main/scala/filodb/prometheus/query/PrometheusModel.scala

```diff
@@ -149,6 +147,12 @@ object PrometheusModel {
     }
   }
 
+  def addVerboseLabelPairs(rvk: RangeVectorKey): Map[String, String] = {
```

Comment:
You are not adding verbose label pairs but creating them. Consider naming method as `makeVerboseLabels`

Reply:
take care of